packages/truffle-debugger/lib/data/selectors/index.js

```diff
@@ -13,6 +13,10 @@ import solidity from "lib/solidity/selectors";
 import * as TruffleDecodeUtils from "truffle-decode-utils";
 import { forEvmState } from "truffle-decoder";
 
+function shallowClone(object) {
+  return Object.assign({}, object);
+}
```

Comment:
How bout `{ ...object }` instead of defining this helper function?

Reply:
Done.